client/js/components/week/week.js

```diff
@@ -1,15 +1,24 @@
 /** @jsx React.DOM */
 var React = require('react');
-var Link = require('react-router-component').Link;
+var WeekHeader = require('./week-header');
+var WeekBoard = require('./week-board');
+var weekData = require('../../weekData');
+var _ = require('underscore');
 
 var Week = React.createClass({
+  getInitialState: function(){
+    return weekData;
+  },
   render: function() {
     return (
       <div>
-        <Link href='/month'>Month View</Link>
+        <WeekHeader />
+        <WeekBoard days={ this.state }/>
```

Comment:
not something that we need to worry about at the moment, but i think that we won't end up passing the entire state through to the WeekBoard, since part of the state will be concerned with the board and part of it will be concerned with the header